src/system/sys_profiler.hh

```diff
@@ -524,6 +638,25 @@ public:
      */
     inline double actual_cumulative_time() const
         { return timers_[actual_node].cumulative_time(); }
+    /**
```

Comment:
The design should not be dependent on unit tests. The better way is to make data members protected and in unit tests create Fixture - auxiliary class derived from the class to test.